2005-10-26  Dmitry V. Levin  <ldv@altlinux.org>

	* src/md5sum.c (main) [!O_BINARY]: Changed default read mode
	back to text, to sync with documentation and for backwards
	compatibility.

--- coreutils-5.92/src/md5sum.c.orig	2005-08-14 15:13:28 +0000
+++ coreutils-5.92/src/md5sum.c	2005-10-26 13:09:51 +0000
@@ -612,6 +612,9 @@ main (int argc, char **argv)
       usage (EXIT_FAILURE);
     }
 
+  if (!O_BINARY && binary < 0)
+    binary = 0;
+
   if (optind == argc)
     argv[argc++] = "-";
 
